mmda/predictors/hf_predictors/vila_predictor.py

```diff
@@ -147,11 +147,7 @@ def predict(self, document: Document) -> List[Annotation]:
         page_prediction_results = []
         for page_id, page in enumerate(tqdm(document.pages)):
 
-            page_width, page_height = document.images[page_id].size
-
-            pdf_dict = convert_document_page_to_pdf_dict(
-                page, page_width=page_width, page_height=page_height
-            ) 
+            pdf_dict = convert_document_page_to_pdf_dict(page)
```

Comment:
Here's where it doesn't seem right to require loading the images just to re-scale the coordinates.

Another way to get the page sizes is from the `Document.pages` I wasn't clear if the page dimensions that come from symbolscraper are the entire page or are just the bounding box that contains all the tokens (page size minus the margins). Having the complete page would be prefereable, IMO.

Reply:
good point.  I think getting it from `.images` at least guarantees that the bbox dimensions are for the whole page.  with deriving them from `.pages` which are of type `SpanGroup`, it's unclear what this bbox should be wrt margins.  i think this will always be ambiguous for `.pages` but at least from `.images` it should be very clear.

hence my preference to go from `.images`

Reply:
I have the same opinion as Kyle!